### PR TITLE
[ES-736] added config for file server mosip logo

### PR DESCRIPTION
**Pull request by kaifk468**

### CODE REVIEW

File: esignet-default.properties [hunk 1]
```diff
@@ -243,6 +243,7 @@ mosip.esignet.cache.expire-in-seconds={'clientdetails' : 86400, \
 
 mosip.esignet.domain.url=https://${mosip.esignet.host}
 mosip.esignet.discovery.issuer-id=${mosip.esignet.domain.url}${server.servlet.path}
+mosip.esignet.file.url=https://${mosip.esignet.host}
```
Comment (vishwa-vyom) [inline esignet-default.properties]: why are we creating this variable ? why not use the mosip.esignet.domain.url ??

File: esignet-default.properties [hunk 2]
```diff
@@ -448,7 +449,7 @@ mosip.esignet.vci.key-values={ 'credential_issuer': '${mosip.esignet.domain.url}
    }},\
   'display': {{'name': 'MOSIP Identity Verifiable Credential', \
                 'locale': 'en', \
-                'logo': {'url': '${mosip.esignet.domain.url}/MOSIP_Logo.png',\
+                'logo': {'url': '${mosip.esignet.file.url}/mosip-logo.png',\
```
Comment (vishwa-vyom) [inline esignet-default.properties]: I see the below is not working.
https://esignet.dev.mosip.net/mosip-logo.png

Also can we avoid the esignet in the URL to show the difference ? Is it not possible to access it using https://api.dev.mosip.net/***

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: esignet-default.properties (hunk 1 above)
Comment (kaifk468) [reply esignet-default.properties]: changed

File: esignet-default.properties (hunk 2 above)
Comment (kaifk468) [reply esignet-default.properties]: now it will be https://api.dev.mosip.net/inji/mosip-logo.png

File: esignet-default.properties [hunk 3]
```diff
@@ -243,6 +243,7 @@ mosip.esignet.cache.expire-in-seconds={'clientdetails' : 86400, \
 
 mosip.esignet.domain.url=https://${mosip.esignet.host}
 mosip.esignet.discovery.issuer-id=${mosip.esignet.domain.url}${server.servlet.path}
+mosip.esignet.file.url=https://${mosip.api.host}
```
Comment (vishwa-vyom) [inline esignet-default.properties]: can you rename this to mosip.esignet.api.url ? Because this has not nothing specific to file.

Comment (kaifk468) [reply esignet-default.properties]: done

**Review by vishwa-vyom — APPROVED** [review]

**Pull request merged**